servicetalk-opentelemetry-http/src/main/java/io/servicetalk/opentelemetry/http/OpenTelemetryHttpServerFilter.java

```diff
@@ -52,21 +59,62 @@
  */
 public final class OpenTelemetryHttpServerFilter extends AbstractOpenTelemetryFilter
```

Comment:
I wonder if we should rename this to `OpenTelemetryHttpServiceFilter` to have proper alignment with the client?

Reply:
ok, will do in a follow up PR